File: 백준/Bronze/10039. 평균 점수/평균 점수.java
import java.io.*;
public class Main{
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int sum = 0;
        for (int i = 0; i < 5; i++) {
            int n = Integer.parseInt(br.readLine());
            if (n < 40) n = 40;
            sum += n;
        }
        System.out.print(sum / 5);
    }
}

File: 백준/Bronze/10039. 평균 점수/평균 점수.cc
#include<iostream>
using namespace std;

int main()
{
	int sum = 0;

	for (int i = 0; i < 5; i++) {
		int val; cin >> val;
		if (val < 40) sum += 40;
		else sum += val;
	}
	cout << sum / 5;
	
	return 0;
}